src/components/LeaderBoard/__tests__/Leaderboard.test.jsx

```diff
@@ -22,7 +22,7 @@ describe('Leaderboard page structure', () => {
     const tableHeader = mountedLeaderboard.find('thead');
     expect(tableHeader.length).toBe(1);
     const tableHeads = tableHeader.find('th');
-    expect(tableHeads.length).toBe(6);
+    expect(tableHeads.length).toBeGreaterThanOrEqual(6);
```

Comment:
Since it is determined that the table will have 7 headers, my suggestion is to modify the test as below. 

```js
  it('should be rendered with 7 Headers', () => {
    const tableHeader = mountedLeaderboard.find('thead');
    expect(tableHeader.length).toBe(1);
    const tableHeads = tableHeader.find('th');
    expect(tableHeads.length).toBe(7);
  });
```